app/controllers/hashtags_controller.rb

```diff
@@ -0,0 +1,10 @@
+class HashtagsController < ApplicationController
+  def index
+    @hashtags = SimpleHashtag::Hashtag.paginate(page: params[:page], per_page: 10)
+  end
+
+  def show
+    @hashtag = SimpleHashtag::Hashtag.find_by_name(params[:hashtag])
+    @hashtagged = @hashtag.hashtaggables if @hashtag
```

Comment:
`@hashtag&.hashtaggables`
http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/